gopro

```diff
@@ -161,6 +180,40 @@ function parse_args {
       -a|--auto-start)
         GOPRO_AUTOSTART=1
         ;;
+      -R|--resolution)
+  	case "$2" in
```

Comment:
Instead of case you should use =~ operator combined with list of acceptable resolutions and then just assign $2 if it is found from the list.

Reply:
sorry, but I am not  used to creating shell scripts, thus I do not understend your suggestion. Please, test and post if result are ok!.